packs/git/rules/github-pr-merge.json

```diff
@@ -9,16 +9,7 @@
         }
     },
 
-    "criteria": {
```

Comment:
This one I wasn't sure about. I got this change on a merge from master... it's easily revert-able. I'm not doing anything with this.